src/Graby.php

```diff
@@ -176,6 +176,7 @@ private function doFetchContent($url)
         $response = $this->httpClient->fetch($url);
 
         $effectiveUrl = $response['effective_url'];
+        $effectiveUrl = str_replace(' ', '%20', $effectiveUrl);
```

Comment:
Because it'll encode all the url and not only the part we want.
For an url like `http://lexpress.io/my awesome image.jpg` it'll generate `http%3A%2F%2Flexpress.io%2Fmy+awesome+image.jpg`.